clang/test/Analysis/system-header.c

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang_analyze_cc1 -analyzer-opt-analyze-headers -analyzer-checker=security.FloatLoopCounter -analyzer-config report-in-system-header=false -isystem %S/system-include -verify %s
```

Comment:
Could you please add another RUN line where the option is set to true? You could use different verify prefixes there (e.g. "-verify=no-report", "-verify=report")